llama_index/storage/storage_context.py

```diff
@@ -100,18 +101,12 @@ def persist(
 
         Args:
             persist_dir (str): directory to persist the storage context
-
         """
-        if fs is not None:
-            docstore_path = concat_dirs(persist_dir, docstore_fname)
-            index_store_path = concat_dirs(persist_dir, index_store_fname)
-            vector_store_path = concat_dirs(persist_dir, vector_store_fname)
-            graph_store_path = concat_dirs(persist_dir, graph_store_fname)
-        else:
-            docstore_path = str(Path(persist_dir) / docstore_fname)
-            index_store_path = str(Path(persist_dir) / index_store_fname)
-            vector_store_path = str(Path(persist_dir) / vector_store_fname)
-            graph_store_path = str(Path(persist_dir) / graph_store_fname)
```

Comment:
I think this breaks windows support?

The original change was trying to fix using `Path` + FSSpec on windows machines. Path was adding the wrong kind of separator.

This change feels like it will have the same issue?

Reply:
You know what, you're right.  Excellent catch, I will fix this

Reply:
Pushed the fix now